scripts/analytics/addon_snapshot.py

```diff
@@ -82,9 +82,9 @@ def get_events(self, date=None):
 
             connected_count = 0
             for node_settings in node_settings_list:
-                if not node_settings.owner.is_bookmark_collection:
-                    connected_count += 1
-
+                connected_count += 1
+                if not node_settings.owner or node_settings.owner.is_bookmark_collection:
```

Comment:
Oh I had this at first! I totally forget why I abandoned it but yeah, should not have abandoned it. Whoops....